finnish.tex

```diff
@@ -0,0 +1,92 @@
+\selectlanguage{finnish}
+\renewcommand{\headline}{\section*{{\LARGE{}Penta$\cdot$game} - Suomi}}
+\renewcommand{\tocent}{Suomi}
+\renewcommand{\translator}{ihmis-suski}
+\renewcommand{\general}{
+  Peli, jonka voi selittää minuutissa, mutta joka säilyy mielenkiintoisena vuosia.
+
+  Kaksinpelin pelaa noin 20 minuutissa. Kolmeen tai neljän pelaajan peli saattaa kestää jopa 90 minuuttia. 
+
+  Pelissä ei käytetä noppaa. Peli sopii 5 vuotiaasta ylöspäin. Pelin tekijä Jan \noun{Suchanek.}
```

Comment:
`Pelin tekijä Jan \noun{Suchanek.}` -> `Pelin keksi Jan \noun{Suchanek.}` but apparently this can be removed.